index.rst

```diff
@@ -44,8 +44,515 @@
 
 .. Add content below. Do not include the document title.
 
+Overview
+========
+
+The site ``api.lsst.codes`` is intended to be a unified front-end for
+API services designed for programmatic consumption to support LSST Data
+Management.  This technote will document how to use the tools that
+SQuaRE has provided to more easily write and deploy these API
+microservices.
+
+In general, a microservice will be a way to extract data from an
+underlying back-end source.  It may be used to extract from multiple sources
+and correlate and aggregate those sources.  It also may be used to
+reformat or reduce that data in a way that is sensible to consume
+programatically.
+
+You may ask, "why not just hit the backend?  What value does the
+microservice provide?"  There are two, somewhat complementary, answers.
+First is that if you put the service behind a unified framework and
+enforce certain standards for discoverability and output format, you can
+make writing tools to query your services a great deal easier.  Second
+is that the first use case for this is in a chatops service, and we
+wanted to make the chatbot, in so far as possible, a presentation-layer
+front end and keep data manipulation and reduction logic out of it.
+This is particularly true since (if we stick with Hubot) the chat logic
+is written in CoffeeScript.  Python is already part of the DM technical
+stack, which CoffeeScript is not, and therefore likely more familiar to
+DM developers.
+
+This document assumes that you will write your microservice in Python 3
+and Flask (assuming you're working in Python, it's nice if it works
+under Python 2 as well), and that you will use the `apikit
+<sqre_apikit>`_ module (source at `GitHub <github_apikit>`_) to provide
+the required metadata routes.
+
+However, there's also a Quick Start Guide if you don't really care *how*
+all this works, as long as it works.
+
+Quick Start Guide (i.e. tl;dr)
+==============================
+
+* Install ``cookiecutter``:
+
+.. code-block:: bash
```

Comment:
I wonder if you can intent these `code-block`s by two spaces to get them rendered in the scope of the list. Not a big deal otherwise.